sdk/security/keyvault/azcertificates/README.md

```diff
@@ -1,6 +1,7 @@
 # Azure Key Vault Certificates client module for Go
 
 * Certificate management (this module) - create, manage, and deploy public and private SSL/TLS certificates
+* Vault administration ([azadmin](https://aka.ms/azsdk/go/keyvault-admin/docs)) - role-based access control (RBAC), settings, and vault-level backup and restore options
```

Comment:
```suggestion
* Managed HSM administration ([azadmin](https://aka.ms/azsdk/go/keyvault-admin/docs)) - role-based access control (RBAC), settings, and vault-level backup and restore options
```
Unsure whether this would make the water more or less muddy...

Reply:
fixed in [d93dca3](https://github.com/Azure/azure-sdk-for-go/pull/20681/commits/d93dca3f6b14b8b4f1d40b620c1c6f7002f5662b)